lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
I went with ``unavailable_on_mobile`` over ``mobile_available`` because it has a ``False`` default, which I feel like is the more natural default for Booleans. Similarly, eventually we will stop using this flag and the natural deprecation process is simply to never set it to true, instead of having set it true forever since we may have client code that checks for it.

I swapped the names at the external API boundary because it seemed goofy to have the mobile API return something called ``unavailable_on_mobile`` and ``only_on_web`` clarified what you could do with it. But I'm happy to make those the same.